### DISCUSSION TITLE
Sketchfab building displays no textures in Mozilla Hubs

### DISCUSSION THREAD

**Discussion by ghost** [mozilla/hubs#4220] [Spoke Questions] [answered]
Hi all.

Context: Absolute newbie. I'm more of a musician and sound designer. I've watched basic Hubs and Blender tutorials. I'm creating a Mozilla Hubs space in which to present a documentary in an interactive space - a kind of virtual video art gallery.

I found a space that I liked on Sketchfab; https://sketchfab.com/3d-models/abandoned-for-product-showcase-vr-gallery-2021-17f16359111d47399a4d22f67e8be9bb

I converted it to a glb in Blender and created a new scene in Spoke, then imported this item. Unfortunately, the textures are no longer visible: https://hubs.mozilla.com/YeYNB2r/tidy-fearless-gala

I suspect I am supposed to "bake in the textures" but none of the tutorials I follow help in any way. I'm a bit out of my depth. If anyone can point me in the right direction,that would be wonderful.

Thank you

Adrienne.

**Comment by j-conrad** [comment, marked as answer]
Sorry you're having this problem. The issue is due to that model using a Specular/Glossiness workflow, which isn't fully supported. You'll need to convert it to a Metallic/Roughness workflow in order to see the textures correctly. You'll notice when viewing the model details in Sketchfab, that it has Spec/Gloss channels:
![image](https://user-images.githubusercontent.com/31968496/117486432-7e6ec380-af1e-11eb-936b-eed1d99e3ca2.png)

Here is some info on the topic that might help:
https://marmoset.co/posts/pbr-texture-conversion/

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, unmark the answer if you need any more information. Thank you!

**Comment by ghost** [comment]
Hi. thanks for your advice.

I think I'm too much of a novice at this stage to undertake the conversion

in any case i am having fun building spaces from the architecture kit in 
Spoke, and will stick to this for now.

On 07.05.21 19:28, j-conrad wrote:
>
> Sorry you're having this problem. The issue is due to that model using 
> a Specular/Glossiness workflow, which isn't fully supported. You'll 
> need to convert it to a Metallic/Roughness workflow in order to see 
> the textures correctly. You'll notice when viewing the model details 
> in Sketchfab, that it has Spec/Gloss channels:
> image 
> <https://user-images.githubusercontent.com/31968496/117486432-7e6ec380-af1e-11eb-936b-eed1d99e3ca2.png>
>
> Here is some info on the topic that might help:
> https://marmoset.co/posts/pbr-texture-conversion/ 
> <https://marmoset.co/posts/pbr-texture-conversion/>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mozilla/hubs/discussions/4220#discussioncomment-711010>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AGYIPIBSEJYNT7X5RKOVMSLTMQPNTANCNFSM44BTFG3A>.
>
-- 
Adrienne Teicher
Sound Designer / Movement Artist
www.hyenaz.com / www.badconscience.com / at.freudianslit.com
